transforms/acceptance/index.js

```diff
@@ -15,5 +15,9 @@ module.exports = function transformer(file, api) {
   // containerLookup to this.owner.lookup
   code = removeImport(j, code, 'embercom/lib/container-lookup');
   code = replaceIdentifier(j, code, 'containerLookup', 'this.owner.lookup');
+
+  // import path for openSettingsMenu
+  code = addImport(j, code, 'openSettingsMenu', 'embercom/tests/helpers/settings-menu');
```

Comment:
Will will only want to import this if the `openSettingsMenu` identifier exists in the current file.

So we should do something like:
```
if (j(code).find(j.Identifier, { name: 'openSettingsMenu' }).length) {
  code = addImport(j, code, 'openSettingsMenu', 'embercom/tests/helpers/settings-menu');
}